addons/web/static/src/scss/form_view.scss

```diff
@@ -475,6 +475,8 @@
             margin-top: 0;
             margin-bottom: 0;
             line-height: inherit;
+            overflow-wrap: anywhere;
+            word-break: normal;
```

Comment:
There is a particular case where this rule is needed as it's the default value and it doesn't seem to be set before ?

```suggestion
```

Reply:
thx! should be good now ^^